### DISCUSSION TITLE
Argocd Image Updater complains "Image seems not to be live in this application, skipping"

### DISCUSSION THREAD

**Discussion by yeskay16** [argoproj/argo-cd#13495] [Q&A]
kubectl logs pods/argocd-image-updater-5dd76865f6-pgg7s -n argocd

time="2023-05-08T20:44:30Z" level=info msg="Starting image update cycle, considering 1 annotated application(s) for update"
time="2023-05-08T20:44:30Z" level=debug msg="Processing application nginx-image-updater-app"
time="2023-05-08T20:44:30Z" level=debug msg="Image 'yeskay16/nginx' seems not to be live in this application, skipping" application=nginx-image-updater-app
time="2023-05-08T20:44:30Z" level=info msg="Processing results: applications=1 images_considered=0 images_skipped=1 images_updated=0 errors=0"

But this application has nginx deployed using the image yeskay16.nginx:v0.1.0. Not sure what I am missing and why argocd complains this ?

**Comment by yeskay16** [comment]
closed

**Reply by naviat** [reply]
could I know what did you do to resolve it?
